odoo/addons/base/migrations/10.0.1.3/pre-migration.py

```diff
@@ -3,9 +3,16 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from openupgradelib import openupgrade
 
+_column_renames = {
+    'res_partner': [
+        ('birthday', None),
+    ],
+}
+
 
 @openupgrade.migrate()
 def migrate(cr, version):
```

Comment:
One question I have now: shouldn't this have `use_env=False` or declare `env` argument?